recipes/pycoalescence/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pycoalescence" %}
+{% set version = "1.2.6rc48" %}
+{% set sha256 = "9edfccc62a4a2e79b83abf0ce19b5f618795253f5ba61995ca474cd1e89abbc0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  detect_binary_files_with_prefix: True
+  skip: True  # [not py3k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not linux]
+    - gxx_linux-64  # [linux]
```

Comment:
The Jinja sintax ``{{ compiler('cxx') }}`` will put the right compiler for the platform. 
So, please don't use ``gxx_linux-64``.

The problem with your ``cmake`` list is because is getting the wrong path to the compiler. And because of that is not finding the libraries and others things. Normally passing the option ``CMAKE_INSTALL_PREFIX=$PREFIX`` and ``CMAKE_PREFIX_PATH=$PREFIX`` is enough. But for some reason your ``CMakeList`` is discarding that, at least your ``installer.py`` script.
I think would be better to compile that by hand using ``cmake`` directly and after that put the ``python`` files inside of the ``site-packages``

Reply:
Okay, thanks for the advice. I think prefix options are getting passed now, but definite it's still finding the wrong compiler from `cmake` (keeps getting gcc 4.8.2 instead of the 7.2 that's from installed `libgcc`). Will try to work out why `gxx_linux-64` works when the Jinja template doesn't.